docs/docs/guides/metadata.md

```diff
@@ -0,0 +1,106 @@
+#  Add metadata to your application
+
+This guide will walk you through the usage of Context Metadata in Keptn.
+
+After reading this guide you will be able to: 
+- add metadata to applications, workloads
+and tasks in two ways.
+- add those same information also in traces
+
+## Introduction
+
+`KeptnAppContext` is a custom resource in Keptn that allows you to add metadata
+and links to traces for a specific application.
+This enables you to enrich your Keptn resources and your traces with additional
+information, making it easier to understand and analyze
+the performance of your applications.
+
+Common usage of this feature may include:
+
+- adding commit id or references related to your GitOps/ CI-CD tooling
+- referring to different stages and actors 
+(ex. who committed this change? In what repo? What is the referred ticket? Was it a hot fix? )
+
+To exploit this feature you need to install Keptn and deploy an application,
+for instance, you can follow [step #1-#3 here](../getting-started/observability.md#step-1-install-keptn)
+
+To visualize traces you will require either Jaeger or Grafana:
```

Comment:
Jaeger will be required in any case in order to be able to collect the traces, as Grafana will use Jaeger as a data source. Maybe we can word it something like:

```suggestion
To collect traces you will require Jaeger.
To visualise and inspect the traces, you can either use the Jaeger UI or Grafana.
```